src/schema/v2/homeView/sections/ShowsForYou.ts

```diff
@@ -1,6 +1,13 @@
 import { ContextModule } from "@artsy/cohesion"
 import { HomeViewSection } from "."
 import { HomeViewSectionTypeNames } from "../sectionTypes/names"
+import { withHomeViewTimeout } from "../helpers/withHomeViewTimeout"
+import { ResolverContext } from "types/graphql"
+import { ShowsConnection } from "schema/v2/me/showsConnection"
+import { emptyConnection } from "schema/v2/fields/pagination"
+import { EventStatusEnums } from "schema/v2/input_fields/event_status"
+
+const SHOWS_TIMEOUT_MS = 6000
```

Comment:
Could we have different timeouts for production and staging? This way, we could optimize the timeouts for the production environment and, at the same time, avoid timeouts in staging that are coming from the environment being 🐌.

Reply:
That's a good point. We have an env/config value that can be set differently between staging and production environments ([HOME_VIEW_RESOLVER_TIMEOUT_MS](https://github.com/artsy/metaphysics/blob/fc6a57aa2d25cb157b68e5620665714a80324b18/src/config.ts#L54)).

Looks like we could check the `PRODUCTION_ENV` var to provide different override values here.

```typescript
const SHOWS_TIMEOUT_MS = config.PRODUCTION_ENV ? 6000 : 10000

// To not specify a custom override in staging and defer to the default.
// I think this might need a slight change to `withHomeViewTimeout` though.
const SHOWS_TIMEOUT_MS = config.PRODUCTION_ENV ? 6000 : undefined
```

Reply:
Addressed here: https://github.com/artsy/metaphysics/pull/6123